tests/Suggestor/Suggestor.test.ts

```diff
@@ -2,16 +2,31 @@
  * @jest-environment jsdom
  */
 import moment from 'moment';
+import * as chrono from 'chrono-node';
 import { getSettings } from '../../src/Config/Settings';
 import type { SuggestInfo } from '../../src/Suggestor';
 import { makeDefaultSuggestionBuilder } from '../../src/Suggestor/Suggestor';
 import { DEFAULT_SYMBOLS } from '../../src/TaskSerializer/DefaultTaskSerializer';
 
 window.moment = moment;
 
+// Set predictable date for all tests in this file
+const mockDate = new Date(moment('2022-07-11 15:00').valueOf());
```

Comment:
I arbitrarily put `15:00` so that the date wasn't midnight. At midnight, `chrono.parseDate` is happy with "Monday" the same day being in the future, so it was failing Mondays in the big test below.